pkg/common/sensor_environment.go

```diff
@@ -13,7 +13,8 @@ func MakeSensorEnvMap(falconSensor falconv1alpha1.FalconSensor) map[string]strin
 
 	// Set proxy values from environment variables if they exist
 	if proxy.Host() != "" {
-		sensorConfig["FALCONCTL_OPT_APH"] = proxy.Host()
+		sensorConfig["FALCONCTL_OPT_APH"] = strings.TrimPrefix(proxy.Host(), "https://")
+		sensorConfig["FALCONCTL_OPT_APH"] = strings.TrimPrefix(proxy.Host(), "http://")
```

Comment:
It would probably be better to drop `s := url.Scheme` from `proxyHostPort` in `pkg/common/proxy.go` instead.... unless you see a reason to keep the transport scheme and strip out the transport later?

Reply:
the URL scheme is needed at least for KPA Proxy config...

Reply:
Probably should check. My initial inclination is that we can remove the `url.Scheme` as KPA has no affect.